Update spatialomics page with Spatial2Galaxy project details (#1481)
usegalaxy-eu/website: index-spatialomics.md

subdomain: spatialomics
---

> # Welcome to the Spatial OMICS Galaxy server

Reviewer: ```suggestion
# Welcome to the Spatial Omics Galaxy Lab
```

Author: I don't think we should call it a lab :) It is not created around a Galaxy community. Unlike labs, this page is only applies to EU server as Galaxy Europe is part of it. I also did not use any Galaxy Labs engine to make this page. It has been written manually.

Reviewer: Then just "Spatial Omics Galaxy"? Or "Galaxy for Spatial Omics"?
Server is kind of an "unfriendly" name, I feel.

Author: or Spatial Omics Galaxy Portal?

Author: Let's decide this in the upcoming meeting

{:.no_toc}

![](/assets/media/spatial_omics_banner.jpg)

> The SpatialOMICS Galaxy server is a hub for all tools related to the analysis of highly multiplexed image-based spatial analysis. This subdomain is a collaborative effort of Spatial2Galaxy and the Goecks Laboratory members. We welcome any suggestions or requests for making tools related to spatial OMICS analysis available on this server. We also welcome contributions to the development of new tools, workflows or trainings!

Reviewer: ```suggestion
The Spatial Omics Galaxy Lab is a hub for all tools related to the analysis of highly multiplexed image-based spatial analysis. This is a collaborative effort by the Spatial2Galaxy project and the Goecks Laboratory members. We welcome any suggestions or requests for making tools related to Spatial Omics analysis available on this Galaxy instance. We also welcome contributions to the development of new tools, workflows or trainings!
```

> The Spatial Omics Galaxy Lab is a hub for all tools related to the analysis of highly multiplexed image-based spatial analysis. This is a collaborative effort by the Spatial2Galaxy project and the Goecks Laboratory members. We welcome any suggestions or requests for making tools related to Spatial Omics analysis available on this Galaxy instance. We also welcome contributions to the development of new tools, workflows or trainings!

Reviewer: Once we decide on a name, we need it to change it here as well.

This server currently features the individual components of the [MCMICRO](https://mcmicro.org/) pipeline, including [BaSiC](https://github.com/ohsu-comp-bio/basic-illumination) for illumination correction, [ASHLAR](https://github.com/ohsu-comp-bio/ashlar) for stitching and registration, [Coreograph](https://github.com/ohsu-comp-bio/UNetCoreograph) to dearray tissue microarrays (TMAs), [UnMICST](https://github.com/ohsu-comp-bio/UnMicst) to create cell or nucleai probability maps, [S3segmenter](https://github.com/ohsu-comp-bio/S3segmenter) for nucleai and cell segmentation and [MCQuant](https://github.com/ohsu-comp-bio/quantification) for feature quantification. More tools for image analysis outside the MCMICRO ecosystem will be added in the future.

> Several spatial transcriptomics data formats and analysis tools have been integrated into Galaxy as part of the [Spatial2Galaxy CMR project](https://elixir-europe.org/how-we-work/scientific-programme/science/cmr/spatial2).

Reviewer: ```suggestion
Several spatial transcriptomics data formats and analysis tools have been integrated into Galaxy as part of the [Spatial2Galaxy ELIXIR Commissioned Service](https://elixir-europe.org/how-we-work/scientific-programme/science/cmr/spatial2).
```

# Content
{:.no_toc}

1. TOC
{:toc}

# Get started

Are you new to Galaxy, or returning after a long time, and looking for help to get started?
Take [a guided tour]({{ page.website }}/tours/core.galaxy_ui){:target="_blank"} through Galaxy's user interface.

# Tools available
Here, we outline all the Galaxy tools that are relevant in the context of spatial omics data analysis.

## Spatial Datatypes and Utilities

> The `SpatialData` datatype and utilities for reading, writing, manipulating and plotting SpatialData objects.

Reviewer: ```suggestion
The `SpatialData` datatype and utilities for reading, writing, manipulating and plotting SpatialData objects:
```

Tool | Description
--- | ---
{% include tool.html id="spatialdata_io" %} | Load common spatial omics formats into SpatialData
{% include tool.html id="spatialdata_operation" %} | Perform operations on SpatialData objects
{% include tool.html id="seurat_create" %} | Create Seurat objects from Xenium spatial data

## Segmentation and Preprocessing

> Tools for segmenting and pre-processing the spatial transcriptomics data into SpatialData objects.

Reviewer: ```suggestion
Tools for segmenting and pre-processing the spatial transcriptomics data into SpatialData objects:
```

Tool | Description
--- | ---
{% include tool.html id="spapros_selection" %} | Selection of marker genes with spapros
{% include tool.html id="spapros_evaluation" %}  | Evaluation of marker genes with spapros
{% include tool.html id="vpt_segment" %} | Vizgen VPT - Segment cells and refine MERSCOPE experiments
{% include tool.html id="vpt_extract" %} | Vizgen VPT - Extract image patches from the mosaic image at the specified coordinates and size

## Spatial Downstream Analysis

> Tools for spatial transcriptomics downstream analysis

Reviewer: ```suggestion
Tools for spatial transcriptomics downstream analysis:
```

Tool | Description
--- | ---
{% include tool.html id="squidpy_spatial" %} | Analyze and visualize spatial multi-omics data with Squidpy
{% include tool.html id="spacexr_rctd" %} | Robust Cell Type Decomposition, or RCTD, is a statistical method for learning cell types from spatial transcriptomics data
{% include tool.html id="spacexr_cside" %} | Cell type-Specific Inference of Differential Expression, or CSIDE, is part of the spacexr R package for learning cell type-specific differential expression from spatial transcriptomics data
{% include tool.html id="liana_methods" %} | Liana ligand_receptor inference and local bivariate spatial metrics for single-cell or spatial data
{% include tool.html id="liana_misty" %} | Liana MISTy learn spatial relationships with multi-view modelling
{% include tool.html id="liana_multi" %} | Liana multi-sample and multi-condition analysis
{% include tool.html id="liana_resource" %} | Liana Resource prior knowledge and ligand-receptor resources
{% include tool.html id="liana_utils" %} | LIANA utility functions for data transformation and preprocessing

## Cell Annotation & Feature Selection

> Tools for marker gene identification and cell-type annotation

Reviewer: ```suggestion
Tools for marker gene identification and cell-type annotation:
```

- [Jeremy Goecks](https://github.com/jgoecks)
- [Cameron Watson](https://github.com/CameronFRWatson)
- [Allison Creason](https://github.com/alliecreason)

> - [Amirhossein Nilchi](https://github.com/nilchia)

Reviewer 2: ```suggestion
- [Amirhossein Naghsh Nilchi](https://github.com/nilchia)
```

- [Khaled Jumah](https://github.com/khaled196)
- [Nicola Soranzo](https://github.com/nsoranzo)
- [Pavankumar Videm](https://github.com/pavanvidem)

# Spatial2Galaxy Partners

| Partners | Description | People involved |
|----------|-------------|-----------------|
| [Erasmus Medical Center](https://www.erasmusmc.nl/en/research/groups/pathology-stubbs) | Implementation and validation of ST toolset | Andrew Stubbs |
| [Berlin Institute of Health at Charité](https://www.hidih.org/research/computational-oncology) | Catalogue ST toolset and use cases | Naveed Ishaque | 
| [University of Bradford](https://www.bradford.ac.uk) | Develop Spatial2Galaxy tutorial suite and deliver training |  Krzysztof Poterlowicz |
| [University of Freiburg](https://galaxyproject.org/freiburg/people/) | Development of Spatial2Galaxy portal, and support tool and workflow development | Björn Grüning |
| [Earlham Institute](https://www.earlham.ac.uk/scientific-group/papatheodorou-group) | Develop Galaxy Demonstrator user case | Irene Papatheodorou |
{:.table.table-striped}

# Supporters

> This Galaxy Lab is a joint project between different groups from the [Spatial2Galaxy ELIXIR Commissioned Service](https://elixir-europe.org/how-we-work/scientific-programme/science/cmr/spatial2){:target="_blank"} and the [Goecks Laboratory](https://www.goeckslab.org/).

Reviewer: ```suggestion
This is a joint project between different groups from the [Spatial2Galaxy ELIXIR Commissioned Service](https://elixir-europe.org/how-we-work/scientific-programme/science/cmr/spatial2){:target="_blank"} and the [Goecks Laboratory](https://www.goeckslab.org/).
```

> This service is a joint project between different groups from the [Spatial2Galaxy](https://elixir-europe.org/how-we-work/scientific-programme/science/cmr/spatial2){:target="_blank"}, an ELIXIR Cellular and molecular research project, [The Goecks Laboratory](https://www.goeckslab.org/).

Reviewer: ```suggestion
This Galaxy Lab is a joint project between different groups from the [Spatial2Galaxy ELIXIR Commissioned Service](https://elixir-europe.org/how-we-work/scientific-programme/science/cmr/spatial2){:target="_blank"} and the [Goecks Laboratory](https://www.goeckslab.org/).
```
